devfiles/java-springboot/devfile.yaml

```diff
@@ -9,44 +9,26 @@ starterProjects:
 components:
   - container:
       name: tools
-      image: maysunfaisal/springbootbuild
+      image: quay.io/eclipse/che-java11-maven:nightly
       memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       mountSources: true
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
-  - container:
-      name: runtime
-      image: maysunfaisal/springbootruntime
-      memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       endpoints:
-        - name: '8080/tcp'
-          targetPort: 8080
-      mountSources: false
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
-  - volume:
-      name: springbootpvc
-      size: 4Gi
+      - name: '8080-tcp'
+        targetPort: 8080
 commands:
   - exec:
       id: devBuild
       component: tools
-      commandLine: "/artifacts/bin/build-container-full.sh"
+      commandLine: mvn install
```

Comment:
It should be `mvn package` as per general maven conventions.. for build purposes.